terraform/aptos-node/aws/variables.tf

```diff
@@ -210,18 +210,6 @@ variable "enable_calico" {
   default     = true
 }
 
-variable "enable_logger" {
```

Comment:
can you replace this with an option to install the new vector daemonset? even if we leave it off for our uses to helm install via some other process, it's still useful to have an option in terraform for 1-click deployment with logging and monitoring